src/dashboard/alerts/alert.py

```diff
@@ -4,7 +4,7 @@
 import hashlib
```

Comment:
could not comment on the above two files under the "packet_analysis" directory but why are they here? Can they be removed since they are both blank?

Reply:
Those were from local testing, I can remove. I didn't see them added.